examples/src/demos/dev/Bloom.js

```diff
@@ -105,8 +106,12 @@ export default () => (
     onCreated={({ gl }) => {
       gl.gammaInput = true
       gl.gammaOutput = true
-      gl.toneMapping = THREE.Uncharted2ToneMapping
```

Comment:
This toneMapping was removed recently in Three.js
https://github.com/mrdoob/three.js/pull/19606

I am following the replacement that was offered in this PR